projects/core/utils/format/pluralize.ts

```diff
@@ -1,9 +1,10 @@
 import {TuiPluralize} from '@taiga-ui/core/types';
 
 /**
- * Выбирает корректную форму множественного числа для отображения количества
- * @param value число
- * @param args массив из трёх форм множественного числа, например ['год', 'года', 'лет']
+ * Selects the correct plural form to display.
```

Comment:
Let's deprecate it and point out that this thing is for Russian language (and they can use an official solution from Angular https://angular.io/api/common/NgPlural)

Reply:
✅

Reply:
> Let's deprecate it and point out that this thing is for Russian language (and they can use an official solution from Angular https://angular.io/api/common/NgPlural)

Reply:
> Let's deprecate it and point out that this thing is for Russian language (and they can use an official solution from Angular https://angular.io/api/common/NgPlural)